platform/platform_windows.go

```diff
@@ -1,25 +1,35 @@
 package platform
 
 import (
-	utils "github.com/DataDog/gohai/windowsutils"
+	"fmt"
+	"golang.org/x/sys/windows/registry"
+	"os"
 )
 
+const registryHive = "SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion"
+const productNameKey = "ProductName"
+const buildNumberKey = "CurrentBuildNumber"
+const majorKey = "CurrentMajorVersionNumber"
+const minorKey = "CurrentMinorVersionNumber"
+
 func getArchInfo() (systemInfo map[string]interface{}, err error) {
 	systemInfo = make(map[string]interface{})
 
-	computerSystem, err := utils.WindowsWMICommand("COMPUTERSYSTEM", "Name", "SystemType")
-	if err != nil {
-		return
-	}
-	systemInfo["hostname"] = computerSystem["Name"]
-	systemInfo["machine"] = computerSystem["SystemType"]
-
-	os, err := utils.WindowsWMICommand("OS", "Version", "Caption")
-	if err != nil {
-		return
-	}
-	systemInfo["kernel_release"] = os["Version"]
-	systemInfo["os"] = os["Caption"]
+	systemInfo["hostname"], _ = os.Hostname()
+	systemInfo["machine"] = "x86_64"
+
+	k, err := registry.OpenKey(registry.LOCAL_MACHINE,
+		registryHive,
+		registry.QUERY_VALUE)
+	defer k.Close()
+
+	systemInfo["os"], _, _ = k.GetStringValue(productNameKey)
+
+	var maj, _, _ = k.GetIntegerValue(majorKey)
+	var min, _, _ = k.GetIntegerValue(minorKey)
+	var bld, _, _ = k.GetStringValue(buildNumberKey)
+	verstring := fmt.Sprintf("%d.%d.%s", maj, min, bld)
```

Comment:
On my system I'm getting `0.0.9600` for the `verstring` instead of `6.3.9600`.

Looked at my registry and here's what I found:
- `CurrentBuildNumber` is `9600`
- the major and minor versions can only be found under `CurrentVersion` (`6.3`)